app/src/modules/settings/routes/data-model/new-collection.vue

```diff
@@ -451,7 +452,7 @@ export default defineComponent({
 				});
 			}
 
-			return fields;
+			return translate(fields);
```

Comment:
We can't translate it in this step, if we'd do that it would save it translated to the database, instead of as the translatable string 🤔

Reply:
Oh didn't realize that. This means we'll need to translate it on the interface?

Reply:
Yeah! That being said, we can only translate where the value is displayed, otherwise we'll have the same problem where the value is overwritten with the translated value 😩